site/profile/manifests/core/icinga_master.pp

```diff
@@ -275,6 +275,38 @@
     api_password  => $api_pwd,
     require       => Mysql::Db[$mysql_director_db],
   }
+  ##Director Daemon
+  user { 'icingadirector':
```

Comment:
Why can't `icingaweb2::module::director::service` be used to manage the director daemon? https://github.com/Icinga/puppet-icingaweb2/blob/299bf56acbb344a7f615acfed62748fb69964aef/manifests/module/director/service.pp

Reply:
Because there is a bug:
Error: Failed to apply catalog: Parameter source failed on File[/etc/systemd/system/icinga-director.service]: Could not understand source [Unit]
Description=Icinga Director - Monitoring Configuration
Documentation=https://icinga.com/docs/director/latest/
Wants=network.target

[Service]
EnvironmentFile=-/etc/default/icinga-director
EnvironmentFile=-/etc/sysconfig/icinga-director
ExecStart=/usr/bin/icingacli director daemon run
ExecReload=/bin/kill -HUP ${MAINPID}
User=icingadirector
SyslogIdentifier=icingadirector
Type=notify

NotifyAccess=main
WatchdogSec=10
RestartSec=30
Restart=always

[Install]
WantedBy=multi-user.target
: bad URI(is not URI?): [Unit]
Description=Icinga Director - Monitoring Configuration
Documentation=https://icinga.com/docs/director/latest/%0AWants=network.target%0A%0A[Service]%0AEnvironmentFile=-/etc/default/icinga-director%0AEnvironmentFile=-/etc/sysconfig/icinga-director%0AExecStart=/usr/bin/icingacli%20director%20daemon%20run%0AExecReload=/bin/kill%20-HUP%20$%7BMAINPID%7D%0AUser=icingadirector%0ASyslogIdentifier=icingadirector%0AType=notify%0A%0ANotifyAccess=main%0AWatchdogSec=10%0ARestartSec=30%0ARestart=always%0A%0A[Install]%0AWantedBy=multi-user.target%0A (file: /etc/puppetlabs/code/environments/IT_2444_director_daemon/modules/systemd/manifests/unit_file.pp, line: 74)